### DISCUSSION TITLE
Chrome Web Store rejections

### DISCUSSION THREAD

**Discussion by dotproto** [ronggang/PT-Plugin-Plus#591] [General]
Hi PT 助手 Plus folks. I just saw an old notification for #318 and wanted to follow up. It appears that the extension has been rejected for keyword spam. I suspect the problem is that the description currently contains a list of supported services:

> * Transmission
> * Synology Download Station
> * µTorrent
> * Deluge
> * qBittorrent v4.1+
> * ruTorrent
> * Flood

From the [Spam section](https://developer.chrome.com/webstore/program_policies#spam) of the Developer Program Policy 

> **Keyword Spam:** We don't allow extensions with misleading, improperly formatted, non-descriptive, irrelevant, excessive, or inappropriate metadata, including but not limited to the extension’s description, developer name, title, icon, screenshots, and promotional images. Developers must provide a clear and well-written description. We also don't allow unattributed or anonymous user testimonials in the product's description.

The [Spam FAQ](https://developer.chrome.com/webstore/spam-faq#how-many%20sites/keywords/brands%20can%20be%20listed%20in%20the%20metadata?) gives a bit more guidance.

> **How many sites/keywords/brands can be listed in the metadata?**
> 
> At most five instances of a keyword. Keywords must be relevant to the extension’s purpose. Do not repeat brand names in the description. When listing supported websites or brands in the description, do not list more than five. To provide a longer list of brands or websites, provide a link that users can refer to or embed the list in one of the extension's promotional screenshots.

At the moment the list of supported services contains 7 entries.

To ge back in the Chrome Web Store you will need to appeal the enforcement action and update your description. I'd recommend only listing the top 3 services you integrate with and providing a URL that lists all integrations. Alternatively, you can list all supported services in a promotional image.

To appeal the takedown please fill out [this form](https://support.google.com/chrome_webstore/contact/one_stop_support) using the following options:

* What can we help you with today?
    * My item
* Please select a sub-category of support
    * My item was warned / removed / rejected
* Please provide us with more information to help you with this request
    * I want to rectify the violation
    * Reference color: Yellow
    * Reference element: Argon
* Do you require more help?
    * Yes

You should now see a form where you can enter information about the extension. In the additional comments section, mention that you were taken down for "keyword stuffing" and would like to submit a new version that addresses the violation.

Good luck!

---

`google translate`

大家好PT助手Plus。我刚刚看到＃318的旧通知，并想跟进。似乎该扩展名已被关键字垃圾邮件拒绝。我怀疑问题在于该描述当前包含支持的服务列表：

> * Transmission
> * Synology Download Station
> * µTorrent
> * Deluge
> * qBittorrent v4.1+
> * ruTorrent
> * Flood

来自开发人员计划政策的[垃圾邮件]（https://developer.chrome.com/webstore/program_policies#spam）部分

> **关键字垃圾邮件：**我们不允许带有误导性，格式不正确，无描述性，无关，过多或不适当的元数据的扩展程序，包括但不限于扩展程序的描述，开发人员名称，标题，图标，屏幕截图，和促销图片。开发人员必须提供清晰且写得很好的描述。我们也不允许在产品说明中使用未经归属或匿名的用户推荐。

[垃圾邮件常见问题解答]（https://developer.chrome.com/webstore/spam-faq#how-many%20sites/keywords/brands%20can%20be%20listed%20in%20the%20metadata？）提供了更多指导。

> **元数据中可以列出多少个网站/关键字/品牌？**
>
>关键字最多五个实例。关键字必须与扩展程序的目的相关。请勿在说明中重复品牌名称。在说明中列出支持的网站或品牌时，请勿列出五个以上的网站。要提供更长的品牌或网站列表，请提供一个链接，用户可以参考该链接或将该列表嵌入到扩展程序的促销屏幕截图之一中。

目前，支持的服务列表包含7个条目。

要返回Chrome网上应用店，您需要对实施措施提出申诉并更新您的描述。我建议仅列出与您集成的前三项服务，并提供列出所有集成的URL。或者，您可以在促销图像中列出所有受支持的服务。

要对删除提出申诉，请使用以下选项填写[此表单]（https://support.google.com/chrome_webstore/contact/one_stop_support）：

* What can we help you with today?
    * My item
* Please select a sub-category of support
    * My item was warned / removed / rejected
* Please provide us with more information to help you with this request
    * I want to rectify the violation
    * Reference color: Yellow
    * Reference element: Argon
* Do you require more help?
    * Yes

现在，您应该看到一个表单，您可以在其中输入有关扩展名的信息。在其他评论部分中，提到您因“关键字填充”而被删除，并希望提交一个新版本来解决违规问题。

祝好运！

**Comment by ronggang** [comment]
@dotproto Thanks for help. I'll try this way.
